lib/rules/string-no-newline/__tests__/index.mjs

```diff
@@ -13,13 +13,17 @@ testRule({
 			description: 'without newline',
 		},
 		{
-			code: "a::before { content: 'one\\ntwo'; }",
+			code: "a::before { content: 'one\\\ntwo'; }",
```

Comment:
This is the correct representation of `backslash + newline` in CSS :

```css
"foo \
bar"
```

With only a single `\` and `\n` it becomes an escaped newline, not a backslash followed by a newline.